tap_hubspot/hubspot.py

```diff
@@ -287,6 +289,20 @@ def get_owners(self):
             data_field=data_field,
             offset_key=offset_key,
         )
+    
+    def get_archived(self, object_type: str):
+        path = f"/crm/v3/objects/{object_type}"
+        data_field = "results"
+        replication_path = ["archivedAt"]
+        offset_key = "after"
+        params = {"limit": 100, "archived": True}
```

Comment:
did you figure out the properties returned here? We would need almost no props apart from the `contact_id`

Reply:
for instance, for contacts, maybe if we just put in contact_id (or whatever the property is) then we won't get a lot of default properties.

Reply:
Should be fixed now